src/common/components/SelectOptionButtons/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { SelectOptionButtonProps } from './models';
+
+import './styles';
+
+export const SelectOptionButtons = ({
+  title,
+  options
+}: SelectOptionButtonProps) => {
+  const [selectedIndex, setIndex] = React.useState(0);
+  return (
+    <div className="select-option-buttons">
+      <div className="select-option-buttons__title">{title}</div>
+      {options &&
+        options.map((option, index) => (
+          <button
+            key={`select-option-button-${index}`}
```

Comment:
using index as a key is a bad practice, since it wouldn't help react optimize reconcile process